apps/zupoll-server/src/routing/routes/pollRoutes.ts

```diff
@@ -341,28 +341,26 @@ export function initPCDRoutes(
           }
         } else if (originalBallotMsg?.length > 0) {
           for (const voteMsg of originalBallotMsg) {
-            const msg = await context.bot?.api.sendMessage(
-              voteMsg.chatId.toString(),
-              generatePollHTML(ballot, allVotes),
-              {
-                reply_to_message_id: parseInt(voteMsg.messageId.toString()),
-                parse_mode: "HTML",
-                // disable_web_page_preview: true,
-                reply_markup: new InlineKeyboard().url(
-                  `See more / Vote`,
-                  `${process.env.BOT_ZUPOLL_LINK}?startapp=${ballot.ballotURL}`
-                )
-              }
-            );
-            if (msg) {
-              // eslint-disable-next-line @typescript-eslint/no-unused-vars
-              const { id, messageId, ...resultsMsg } = voteMsg;
-              await saveTgMessage(
-                msg,
-                resultsMsg.ballotId,
-                MessageType.RESULTS
+            try {
+              const msg = await context.bot?.api.sendMessage(
+                voteMsg.chatId.toString(),
+                generatePollHTML(ballot, allVotes),
+                {
+                  reply_to_message_id: parseInt(voteMsg.messageId.toString()),
+                  parse_mode: "HTML",
+                  // disable_web_page_preview: true,
+                  reply_markup: new InlineKeyboard().url(
+                    `See more / Vote`,
+                    `${process.env.BOT_ZUPOLL_LINK}?startapp=${ballot.ballotURL}`
+                  )
+                }
               );
-              logger.info(`Updated DB with RESULTS`);
+              if (msg) {
+                await saveTgMessage(msg, voteMsg.ballotId, MessageType.RESULTS);
+                logger.info(`Updated DB with RESULTS`);
+              }
+            } catch (error) {
+              logger.error(`GRAMMY ERROR`, error);
```

Comment:
fyi zupoll has rollbar now - potentially worth logging caught errors to that

Reply:
I've added Rollbar logging to the errors in this function